projects/plugins/crm/readme.txt

```diff
@@ -1,7 +1,7 @@
 === Jetpack CRM - Clients, Leads, Invoices, Billing, Email Marketing, & Automation ===
 Contributors: automattic, kallehauge, cleacos, diegogarciarodrigues, bradshawtm, wpkaren, robertf4, woodyhayday, mikemayhem3030
-Tags: CRM, Invoice, Woocommerce CRM, Clients, Lead Generation, contacts, customers, billing, email marketing, Marketing Automation, contact form, automations
-Tested up to: 6.8
+Tags: CRM, Woocommerce CRM, Client Portal, Marketing Automation, Lead Generation
+Tested up to: 6.8.1
```

Comment:
This shouldn't hurt anything, but minor versions are ignored:

> This field ignores minor versions, as plugins shouldn’t break with a minor update.

https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/

On the WP.org plugin page it'll show whatever the latest minor is for the current tested version.